Elevel/Elevel.Api/Startup.cs

```diff
@@ -32,6 +32,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAutoMapper(typeof(TestQuestionProfile));
+            services.AddAutoMapper(typeof(ApplicationUserProfile));
```

Comment:
you don't need to specify any other type, it will work from 34 row and get everything from Assembly as I remember

Reply:
If these profiles founds in different Assemblies yhou need specify all these assembly but if we use the same assembly for profily it's not nessessary specify this twice.
https://docs.automapper.org/en/stable/Dependency-injection.html#asp-net-core